godot_project/editor/controls/dockers/workspace_docker/simulations_docker/simulations_docker_controls/simulation_settings_panel.tscn

```diff
@@ -107,16 +109,95 @@ popup/item_0/text = "Show user defined files"
 popup/item_0/checkable = 1
 popup/item_0/id = 0
 
-[node name="ExtensionsInfoLabel" parent="VBoxContainer" instance=ExtResource("2_i1oqk")]
+[node name="ExtensionsInfoLabel" parent="." instance=ExtResource("2_i1oqk")]
 unique_name_in_owner = true
 layout_mode = 2
 text = "[center][pulse start=4 length=14 freq=0.5 sat=0.8 val=0.8 connected=1]ℹ[/pulse] MSEP.one Extensions includes unvalidated estimated values[/center]"
 message = &"MSEP.one Extensions includes unvalidated estimated values"
 effect = "pulse"
 
-[node name="UserForcefieldInfoLabel" parent="VBoxContainer" instance=ExtResource("2_i1oqk")]
+[node name="UserForcefieldInfoLabel" parent="." instance=ExtResource("2_i1oqk")]
 unique_name_in_owner = true
 layout_mode = 2
 text = "[center][pulse start=4 length=14 freq=0.5 sat=0.8 val=0.8 connected=1]ℹ[/pulse] Force Field files defined by user are in use. Use with care under your own risk[/center]"
 message = &"Force Field files defined by user are in use. Use with care under your own risk"
 effect = "pulse"
+
+[node name="AdvancedOptionsCheckBox" type="CheckBox" parent="."]
+layout_mode = 2
+focus_mode = 0
+theme_override_font_sizes/font_size = 22
```

Comment:
:warning: Was the font size override explicitly requested?
It looks out of place and no over checkbox does that

Reply:
I did this to match the font size with **HeaderSmall** font variation of **Label** class

Reply:
I changed properties overrides for a new type variation **HeaderSmallCheckBox**

Reply:
Oh right, because it's meant to be a title. Would have been nice if the checkbox was on the right but I don't think that's an option.